catherine_feijoo/happitails/main.rb

```diff
@@ -0,0 +1,86 @@
+require_relative 'lib/animal'
+require_relative 'lib/client'
+require_relative 'lib/shelter'
+require_relative 'seeds'
+
+# print "what would you like your shelter name to be?   "
+# name = gets.chomp.capitalize
+# shelter = Shelter.new( name )
+
+include Seeds
+
+shelter = Seeds.get_shelter
+
+input = ""
+while input != "q"
+	puts "*********************************"
+	puts "choose from the following options"
+	puts "  create an animal (ca)"
+	puts "  create a client  (cc)"
+	puts "  view all animals (va)"
+	puts "  view all clients (vc)"
+	puts "  adopt animal     (aa)"
+	puts "  return animal    (ra)"
+	puts "  quit             (q)"
+
+	input = gets.chomp.downcase
+	if input == "ca"
+		print "enter animal name: "
+		animal_name = gets.chomp.capitalize
+		print "enter animal species: "
+		animal_species = gets.chomp
+		new_animal = Animal.new( animal_name, animal_species )
+		shelter.add_animal( new_animal )
+
+	elsif input == "cc"
+		print "enter client name: "
+		client_name = gets.chomp.capitalize
+		print "enter client age: "
+		client_age = gets.chomp.to_i
+		new_client = Client.new( client_name, client_age )
+		shelter.add_client( new_client )
+
+	elsif input == "va"
+		shelter.show_animals
+
+	elsif input == "vc"
+		shelter.show_clients
+
+	elsif input == "aa"
+		print "which client would like to adopt a pet? "
+		client_name = gets.chomp.capitalize
+		if shelter.has_client?( client_name )
+			current_client = shelter.get_client( client_name )
+			print "which animal would #{ client_name } like to adopt? "
+			animal_name = gets.chomp.capitalize
```

Comment:
Nice. I (and everyone that i advised on this) made the animals collection an array but i suppose that it is more user friendly than doing it by array index. then again... what if there are two dogs named 'Pookie'